src/Microsoft.Kusto.ServiceLayer/HostLoader.cs

```diff
@@ -91,7 +90,7 @@ private static void InitializeRequestHandlersAndServices(ServiceHost serviceHost
             QueryExecutionService.Instance.InitializeService(serviceHost);
             serviceProvider.RegisterSingleService(QueryExecutionService.Instance);
 
-            ScriptingService.Instance.InitializeService(serviceHost, scripter, dataSourceFactory);
```

Comment:
No longer creating connection within ScriptingService so we need the ConnectionService to be able to access existing Kusto connections